textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -141,6 +146,23 @@ public static ITuple getSpanTuple(List<IField> fieldList, List<Span> spanList, S
                 .map(attr -> attr.getFieldName())
                 .collect(Collectors.toList());
     }
+    
+    /**
+     * Create a new schema object, with "_ID" attribute added to the front.

```

Comment:
Be more clear in the comment, e.g., add the `_id` attribute to the beginning of the schema only if the schema didn't include attribute before.